README.md

```diff
@@ -27,12 +27,26 @@ Cozy's apps use a standard set of _npm scripts_ to run common tasks, like watch,
 
 ### Fixtures
 
-You can import fixtures to quickly deal with data:
+You can import fixtures to quickly deal with data.
+
+First, make sure you have the latest version of ACH installed
+
+```bash
+$ yarn global add cozy-ach
+```
+
+Finally, run the following command to import fixtures:
 
 ```sh
 $ yarn fixtures
 ```
 
+Alternatively, if you do not wish to work on the default `http://cozy.localhost:8080` URL, please use the following command (the `-x` option is to disable the dry-run mode):
```

Comment:
Is the `-x` also necessary for the `yarn fixtures `? It looks odd that the default is with dry-run enabled

Reply:
Yes, it is dry-run by default, at least when you provide an URL, though I don't know why it was chosen to be that way. Should we invert that? I guess the dry mode is set at the global ACH level

Reply:
> Les remarques sont pertinentes, mais on devrait plutôt mettre de la doc dans les outils concernés, car elles sont potentiellement vraies pour toutes les app

perso je suis un peu perdu sur comment répartir les choses, en tout cas ce que je peux dire c'est que tout ce que j'ai ajouté est indispensable pour avoir les fixtures sur une instance nip.io